Classes/Command/AbstractCommand.php

```diff
@@ -15,13 +15,27 @@
 use B13\Make\Environment\Variables;
 use B13\Make\Exception\EmptyAnswerException;
 use B13\Make\Exception\InvalidPackageNameException;
+use B13\Make\PackageResolver;
 use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use TYPO3\CMS\Core\Package\PackageInterface;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
 
 /**
  * Abstract command with basic functionalities
  */
 abstract class AbstractCommand extends Command
 {
+    /** @var SymfonyStyle */
```

Comment:
Since we do the `$io` initialization here, we should remove duplicated code in `SimpleComponentCommand` (property as well as initialization. Calling `parent::initialize($input, $output)` there should be sufficient).

Reply:
Done